GSA/accessibility-for-teams
# Update node to V18

This PR will update the `.nvmrc` file to node V18 this is due to V16 hitting end of life.

testing:

1. pull down branch 
2. run `npm install` and `npm run start` and ensure site is running correctly
